### PR TITLE
Never reconnect a cockpit transport

### PR DESCRIPTION
**Pull request by stefwalter**
We have to refresh the page anyway in order to get things back to a consistent state, even if we avoid logging in again.

Opening the transport again automatically simply leads to races during shutdown, reboot, or problem conditions.

### CODE REVIEW

**Comment by petervo** [comment]
The relogin command in tests seems broken.

**Comment by stefwalter** [comment]
You're right. Need to think about this more.

**Comment by stefwalter** [comment]
Failures look legit. Debugging.

File: pkg/base1/cockpit.js [hunk 1]
```diff
@@ -66,9 +58,11 @@ var filters = [ ];
 
 var have_array_buffer = !!window.ArrayBuffer;
 
-var origin = window.location.origin;
-if (!origin) {
-    origin = window.location.protocol + "//" + window.location.hostname +
+var transport_origin = window.location.origin;
+var transport_options = { };
```
Comment (dperpeet) [inline pkg/base1/cockpit.js]: is this actually used anywhere?

Comment (stefwalter) [inline pkg/base1/cockpit.js]: Nope. Will remove.

**Comment by dperpeet** [comment]
Aside from the one comment, this looks good to merge.

File: pkg/base1/cockpit.js (hunk 1 above)
Comment (stefwalter) [inline pkg/base1/cockpit.js]: Done.

**Pull request closed without merging**